### PR TITLE
psgi_loader: fix a one off bug in XS_input_seek()

### PR DESCRIPTION
**Pull request by ikruglov**

### CODE REVIEW

File: plugins/psgi/psgi_loader.c [hunk 1]
```diff
@@ -12,7 +12,7 @@ XS(XS_input_seek) {
 	struct wsgi_request *wsgi_req = current_wsgi_req();
 
         psgi_check_args(1);
```
Comment (unbit) [inline plugins/psgi/psgi_loader.c]: here we should check for 2 args too (instead of 1)

Comment (ikruglov) [inline plugins/psgi/psgi_loader.c]: why would one check params which are not used?

Comment (avar) [inline plugins/psgi/psgi_loader.c]: What this does is make sure you have at least 1 (argument), i.e. the object (0) and the position.

Now you can just call $input-seek() and it won't error out.

@unbit: The full interface is $input->seek(0, [$whence]), where $whence is SEEK_{SET,CUR,END}. See "perldoc -f seek", any chance of supporting that?

I don't need it for anything, just curious.

**Comment by ikruglov** [comment]
Please merge the patch! It fixes real bug in psgi plugin.

**Comment by unbit** [comment]
Oh sorry, did not notice you already fixed the check_args value

**Comment by ikruglov** [comment]
Thanks!

**Pull request merged**